health/writer/writer.go

```diff
@@ -0,0 +1,46 @@
+/*
+Package writer implements health data writer
+
+It listens a channel provided by health manager and pushes data to defined destination
+*/
+package writer
+
+import (
+	"github.com/zenoss/zenoss-go-sdk/health/log"
+	"github.com/zenoss/zenoss-go-sdk/health/target"
+)
+
+// HealthWriter provides an ability to forward health data from manager to destination
+type HealthWriter interface {
+	// Start should be run in goroutine.
+	// It listens for healthIn channel and sends data to configured destination
+	Start(healthIn <-chan *target.Health)
+}
+
+// New creates a new HealthWriter instance.
+// Destination should be initialized and passed here as a parameter
+func New(dest Destination) HealthWriter {
+	return &writer{destination: dest}
+}
+
+type writer struct {
+	// it can be a list of Destinations in future if needed
```

Comment:
Yes, we do need to support multiple destinations

Reply:
Multiple destinations support added: writer handles a list of destinations